### PR TITLE
Script covering cyclic removing due to appIconsAmountToRemove

### PR DESCRIPTION
**Pull request by aarotar**
@vvvakulenko, @dboltovskyi, please review

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@aarotar , I think target branch to commit should be not smartdevicelink:develop, but smartdevicelink:feature/extended_policy_functionality

**Comment by vvvakulenko** [comment]
@aarota, please reference the script to TestSet, as it was stated in https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=308516164

File: test_scripts/AppIconsFolder/ATF_P_Cyclically_Remove_AppIconsAmountToRemove.lua [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  local status = true
+  commonFunctions:userPrint(32, "Content of storage folder: " ..tostring("\n" ..ListOfFilesInStorageFolder))
+  local iconFolderPath =  config.pathToSDL .. tostring("Icons/")
+  local applicationFileToCheck = iconFolderPath .. RAIParameters.appID
+  local applicationFileExistsResult = checkFilePresent(applicationFileToCheck)
+  if applicationFileExistsResult ~= true then
+    commonFunctions:userPrint(31, tostring(RAIParameters.appID) .. " icon is absent")
+    status = false
+  end
+  for i=1, 2, 1 do
```
Comment (vvvakulenko) [inline test_scripts/AppIconsFolder/ATF_P_Cyclically_Remove_AppIconsAmountToRemove.lua]: @aarotar , could be simplified to: for i=1, 3 do

File: test_scripts/AppIconsFolder/ATF_P_Cyclically_Remove_AppIconsAmountToRemove.lua [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  StopSDL()
+end
+
+function Test.Postcondition_deleteCreatedIconsFolder()
+  assert(os.execute( "rm -rf " .. testIconsFolder))
+end  
+
+function Test.Postcondition_restoreDefaultValuesInIni()
+  commonFunctions:SetValuesInIniFile("AppIconsFolder%s-=%s-.-%s-\n", "AppIconsFolder", 'storage')
+  commonFunctions:SetValuesInIniFile("AppIconsFolderMaxSize%s-=%s-.-%s-\n", "AppIconsFolderMaxSize", 104857600)
```
Comment (dboltovskyi) [inline test_scripts/AppIconsFolder/ATF_P_Cyclically_Remove_AppIconsAmountToRemove.lua]: @aarotar, Default value for 'AppIconsFolderMaxSize' parameter is 1048576

**Comment by aarotar** [comment]
@dboltovskyi, updated in 21c3e5f

**Pull request merged**